modules/cnv-understanding-live-migration.adoc

```diff
@@ -5,16 +5,14 @@
 [id="cnv-understanding-live-migration_{context}"]
 = Understanding live migration
 
-Live migration is the process of moving a running virtual machine instance to 
-another node in the cluster without interruption to the virtual workload or 
-access. This can be a manual process, if you select a virtual machine instance 
-to migrate to another node, or an automatic process, if the 
-virtual machine instance has a `LiveMigrate` eviction strategy and the node on 
-which it is running is placed into maintenance. 
+Live migration is the process of moving a running virtual machine instance to
+another node in the cluster without interrupting the virtual workload.
 
-[IMPORTANT]
-====
-Virtual machines must have a PersistentVolumeClaim (PVC) with a shared 
-ReadWriteMany (RWX) access mode to be live migrated.
-====
+This process is manual if you select a virtual machine instance
+to migrate to another node. The process is automatic if the
+virtual machine instance has a `LiveMigrate` eviction strategy and the node on
+which it is running is placed into maintenance.
 
+[IMPORTANT]
```

Comment:
Needs the appropriate markup. Also "ReadWriteMany".
Might be easier to 'git checkout -- cnv-understanding-live-migration.adoc` if there are no changes to the module.

Reply:
Reverted this to original by making appropriate edits. Was not able to figure out how to remove the changed file from GitHub. Doing git checkout -- would have prior to commit but not sure how to revert if once it's in GitHub. if you have info, let me know.